app/services/journeyAnswers/IncomeAnswersService.scala

```diff
@@ -70,9 +70,10 @@ class IncomeAnswersServiceImpl @Inject() (repository: JourneyAnswersRepository,
     val storageAnswers = IncomeStorageAnswers.fromJourneyAnswers(answers)
 
     val upsertBody = CreateAmendSEAnnualSubmissionRequestBody.mkRequest(
-      Some(AnnualAdjustments.empty.copy(includedNonTaxableProfits = answers.notTaxableAmount, outstandingBusinessIncome = answers.otherIncomeAmount)),
-      Some(AnnualAllowances.empty.copy(tradingIncomeAllowance = answers.tradingAllowanceAmount)),
-      None
+      annualAdjustments = Some(
+        AnnualAdjustments.empty.copy(includedNonTaxableProfits = answers.notTaxableAmount, outstandingBusinessIncome = answers.otherIncomeAmount)),
+      annualAllowances = Some(AnnualAllowances.empty.copy(tradingIncomeAllowance = answers.tradingAllowanceAmount)),
+      annualNonFinancials = None
```

Comment:
As discussed offline, we decided to create an epic and putting a new story to go through testing once QA is back. This Nones are suspicious to me.